### PR TITLE
add the functionality to change task/sensitive label for an image dat…

### PR DESCRIPTION
**Pull request by olliethomas**
…aset

### CODE REVIEW

File: ethicml/vision/data/image_dataset.py [hunk 1]
```diff
@@ -55,6 +56,44 @@ def __init__(
...
+                self.additional_columns,
+                pd.DataFrame(self.s.detach().cpu().numpy(), columns=[self.s_label]),
+            ],
+            axis=1,
+        )
+        self.possible_labels.append(self.s_label)
+        s = torch.as_tensor(self.additional_columns[label].to_numpy()).unsqueeze(-1)
+        self.s = (s + 1) // 2 if s.min() == -1 and s.max() == 1 and len(s.unique()) == 2 else s
+        self.additional_columns = self.additional_columns.drop(label, axis=1)
+        self.possible_labels.remove(label)
```
Comment (thomkeh) [inline ethicml/vision/data/image_dataset.py:95]: please de-duplicate this code

**Review by thomkeh — CHANGES_REQUESTED** [review]

File: ethicml/vision/data/image_dataset.py (hunk 1 above)
Comment (olliethomas) [reply ethicml/vision/data/image_dataset.py]: This is now de-duped, but it could probably still be nicer

**Review by thomkeh — APPROVED** [review]
